app/src/main/res/values/strings.xml

```diff
@@ -746,4 +746,5 @@
   <string name="screen_teleconsult_share_prescription_image_using">Share image using</string>
   <string name="screen_teleconsult_share_prescription_title">Prescription</string>
   <string name="screen_teleconsult_share_prescription_instructions_empty" translatable="false">-</string>
-</resources>
+  <string name="screen_teleconsult_share_prescription_image_saved">%1$s is saved to Downloads</string>
```

Comment:
Can we get this string copy reviewed by someone from the design team?

Reply:
Updated the toast message.